templates/conf/kubernetes/containers.conf.erb

```diff
@@ -7,7 +7,6 @@
   exclude_path "#{ENV['FLUENT_CONTAINER_TAIL_EXCLUDE_PATH'] || use_default}"
   read_from_head true
   <parse>
-    @type "#{ENV['FLUENT_CONTAINER_TAIL_PARSER_TYPE'] || 'json'}"
-    time_format "#{ENV['FLUENT_CONTAINER_TAIL_PARSER_TIME_FORMAT'] || '%Y-%m-%dT%H:%M:%S.%NZ'}"
+    @include ../tail_container_parse.conf
   </parse>
```

Comment:
tail_container_parse.conf already uses `<parse>` directive. We should just include it here:

```
@include ../tail_container_parse.conf
```

Reply:
Sorry, I failed to amend the commit before pushing it.
I've fixed it.